### DISCUSSION TITLE
openfisca france

### DISCUSSION THREAD

**Discussion by DRCOMPUTER60290** [community/community#145908] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Bonjour Madame , Monsieur ,
je vous contacte afin de solliciter votre aide .
j'ai le projet de monter un site web (faciltaides.fr qui est déjà en ligne) avec un simulateur d'aide pour la caf (les principales aides) et ensuite concernant ma prime rénov  (dans le futur). (en france donc avec openfisca france).
J'aimerais utiliser l'api openfisca afin de mettre en place ce simulateur sur mon site développer sous wordpress et hébergé sous ionos , j'ai aussi un vps (sous ubuntu 24) pour l'hébergé. je souhaiterai juste que quelqu'un m'aide s'il vous plait.
Merci de votre aide.

**Comment by ebndev** [comment]
Hi @DRCOMPUTER60290, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
